DOCPLAN-156: CQA fixes for node maintenance docs (#109711)
openshift/openshift-docs: modules/virt-delete-failed-node-vm-failover.adoc

// Module included in the following assemblies:
//
// * virt/nodes/virt-eviction-strategies.adoc

:_mod-docs-content-type: PROCEDURE
[id="virt-delete-failed-node-vm-failover_{context}"]
= Delete a failed node to trigger virtual machine failover

[role="_abstract"]
If a node fails and machine health checks are not deployed on your cluster, virtual machines (VMs) with `runStrategy: Always` configured are not automatically relocated to healthy nodes. To trigger VM failover, you must manually delete the `Node` object. The following procedure deletes a node from an {product-title} cluster running on bare metal.

.Prerequisites

* A node where a virtual machine was running has the `NotReady` condition.
* The virtual machine that was running on the failed node has `runStrategy` set to `Always`.
* You have installed the {oc-first}.

.Procedure

. Mark the node as unschedulable:
+
[source,terminal]
----
$ oc adm cordon <node_name>
----

. Drain all pods on the node:
+
[source,terminal]
----
$ oc adm drain <node_name> --force=true
----
+

> This step might fail if the node is offline or unresponsive. Even if the node does not respond, the node might still be running a workload that writes to shared storage. To avoid data corruption, power down the physical hardware before you proceed.

Reviewer: ```suggestion
This step might fail if the node is offline or unresponsive. Because the node has access to shared storage, power down the physical hardware to avoid data corruption.
```

Should this admonition be stronger than a suggestion? Should it say 'you must'?

Author: I think for the scope of the CQA fixes it's OK as is for now.
